README.md

```diff
@@ -77,9 +77,10 @@ List of available permissions methods:
  * `isProd` - test if current environment is production (prod-beta and prod-stable)
  * `isBeta` - test if current environment is beta (ci-beta, qa-beta and prod-beta)
  * `hasPermissions` - test if current user has rbac role permissions ['app:scope:permission']
- * `apiRequest` - call custom API endpoint to test if the item should be displayed
+ * `apiRequest` - call custom API endpoint to test if the item should be displayed. Expects `true`/`false` response. If the promise receives an error, the item won't be displayed.
```

Comment:
I'd also add that consumers can pass in `selector` if the response is a bit more complicated with description of lodash's `get` function.